core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -174,6 +174,17 @@ package object config {
   private[spark] val EVENT_LOG_CALLSITE_LONG_FORM =
     ConfigBuilder("spark.eventLog.longForm.enabled").booleanConf.createWithDefault(false)
 
+  private[spark] val EVENT_LOG_ENABLE_ROLLING =
+    ConfigBuilder("spark.eventLog.rollLog")
+      .doc("Whether rolling over event log files is enabled.")
+      .booleanConf.createWithDefault(false)
+
+  private[spark] val EVENT_LOG_ROLLED_EVENT_LOG_MAX_FILE_SIZE =
+    ConfigBuilder("spark.eventLog.rollLog.maxFileSize")
+      .doc("The max size of event log file to be rolled over, in KiB unless otherwise specified.")
```

Comment:
Uh, I'm not sure this has been respected, please search "in [a-zA-Z]+ unless otherwise specified." with regex. Most of conf docs using bytesConf mention this. Would they need to be changed as well?

Reply:
You don't need to change existing ones, just don't mention units in docs when adding new configs. Users are expected to provide the unit in the config value.

Reply:
OK got it. That makes sense if we always want users to provide the unit. Thanks for explaning!